app/services/flow/steps/step.rb

```diff
@@ -0,0 +1,9 @@
+module Flow
+  module Steps
+    def self.urls
+      Rails.application.routes.url_helpers
+    end
+
+    Step = Struct.new(:path, :forward, :check_answers)
```

Comment:
we could use struct with keyword argument. perhaps slightly clearer

```suggestion
    Step = Struct.new(:path, :forward, :check_answers, keyword_init: true)
```

This allows nil args to be supplied too
```ruby
> Step.new(path: 'test', forward: 'forward')
 => #<struct Step path="test", forward="forward", check_answers=nil>
```

example usage: 
```ruby
module Flow
  module Steps
    CheckClientDetailsStep = Step.new(
      path: ->(application) { urls.providers_legal_aid_application_check_client_details_path(application) },
      forward: :received_benefit_confirmations,
    )
  end
end
```

Reply:
👍 changed to use keyword arguments